### DISCUSSION TITLE
對高中生最大的機器人比賽

### DISCUSSION THREAD

**Discussion by Chenszuhong** [1101-GS3073-C/Python-Community-GS3073-C#490] [General]
我要介紹的是FRC，全名是First Robotic Competition，我自己有親自參加過，雖然過程很辛苦，但是可以學到很多東西，像是視覺辨識或是自動化的程式都可以在這比賽中學到https://www.firstinspires.org/robotics/frc

**Comment by NcuJerryKao** [comment]
我們高中也有同學參加過 還要去洛杉磯比賽 著實蠻辛苦的
不過他們那一組卻有人躺分爽賺 算了 那又是另一個故事了

**Comment by williamyen0623** [comment]
不管是什麼比賽，參加過程肯定都會很辛苦，但能在過程中學到很多，不只是技術，還能觀察其他人的開發思維，可說是促進自己學習的一大途徑

**Comment by Ben6uo** [comment]
但最重要的是機手跟編程的配合吧

**Comment by HsuChen88** [comment]
我室友是FRC王者，有需要可以洽詢。

**Comment by Yilin20021016** [comment]
VEX都是一個很好玩的機械人比賽

**Comment by Raywei0214** [comment]
我以前有參加過格鬥機器人比賽，但我們的機器人不是格鬥專用的，是那種超級簡單的16軸馬達機器人，參數調老半天走路都有困難，比賽當天當然是直接被揍飛。想到就鼻酸

**Comment by larry40305** [comment]
我高中的時候都不知道我未來要幹嘛，現在也是

**Comment by 710388** [comment]
唯一支持aigo 佛系公司 午餐還有冰棒可以吃
